Dockerfile

```diff
@@ -206,6 +201,14 @@ RUN curl -L https://github.com/shenwei356/csvtk/releases/download/v0.24.0/csvtk_
 # Add seqkit
 RUN curl -L https://github.com/shenwei356/seqkit/releases/download/v2.2.0/seqkit_linux_amd64.tar.gz | tar xz --no-same-owner -C /usr/local/bin
 
+# Add gofasta (for ncov/Pangolin)
+RUN curl -fsSL https://github.com/virus-evolution/gofasta/releases/download/v0.0.6/gofasta-linux-amd64 \
+  -o /usr/local/bin/gofasta
+
```

Comment:
Do we not need to retain the `chmod +x`?

Reply:
Oh I see. Yes, it needs retaining, but it's omission wasn't noticed because the the change wasn't tested in isolation but with 472f556.

Reply:
Consolidated the `chmod` changes into 48e4ca2a895de6c54b4e7fbcf5321c28ddaf4074.